models/issue.go

```diff
@@ -597,7 +596,7 @@ func (issue *Issue) ReadBy(ctx context.Context, userID int64) error {
 		return err
 	}
 
-	return setIssueNotificationStatusReadIfUnread(db.GetEngine(db.DefaultContext), userID, issue.ID)
+	return setIssueNotificationStatusReadIfUnread(db.DefaultContext, userID, issue.ID)
```

Comment:
```suggestion
	return setIssueNotificationStatusReadIfUnread(ctx, userID, issue.ID)
```
Should pass the context to `UpdateIssueUserByRead` too.